docs/sdk/api/apiRequestBuilder.md

```diff
@@ -14,27 +14,31 @@ npm install --save @commercetools/api-request-builder
 <script>// global: CommercetoolsApiRequestBuilder</script>
 ```
 
-## `createRequestBuilder(customServices)`
+## `createRequestBuilder(projectKey, customServices)`
```

Comment:
@wizzy25 and I actually discussed this format, but can't remember why we didn't go with it. We can refactor it. Looks better-managed params